website/blog/2020-07-21-62updates.md

```diff
@@ -0,0 +1,48 @@
+* [ReactContext](https://microsoft.github.io/react-native-windows/docs/ireactcontext-api) : Given to native modules to communicate with other native modules, views, application, and the react-native instance
+* [ReactPropertyBag](https://microsoft.github.io/react-native-windows/docs/ireactpropertybag-api) : Provides a thread-safe property storage
+* [XamlUIService](https://microsoft.github.io/react-native-windows/docs/xamluiservice-api) : Provides access to XAML UI specific functionality. See [Native UI components](https://microsoft.github.io/react-native-windows/docs/view-managers) for more details.
+* [RedBoxHandler](https://microsoft.github.io/react-native-windows/docs/iredboxhandler-api) : Provides an extension point to allow custom error handling within the react-native instance
+
+## Alignment with core
+As part of our ongoing alignment with React Native core we have added Windows support for several new APIs introduced in ReactNative 0.62. Few callouts here:
+   * [Appearance](https://reactnative.dev/docs/appearance) module is now supported on Windows. You can use this module to get information on the user's dark/light theme preferences and apply the right styling as needed. Windows previously 
+   * [focusable](https://reactnative.dev/docs/view#focusable) prop is now supported on `View` and `TextInput`. You can use this prop to indicate whether a component can be focusable (participate in tab order) when navigating with keyboard. Windows previously had an `acceptsKeyboardFocus` prop that is now deprecated in favor of aligning with React Native core. 
+   * **RTL Support in Windows** : We have added support to detect when an app is running in RTL locale and automatically apply the RTL [LayoutDirection](https://reactnative.dev/docs/flexbox#layout-direction). In addition, Windows now supports `forceRTL` and `allowRTL` modules.
+
+![rtl](assets/rtl.gif)
+    
+## WinUI 3
+The Windows UI Library ([WinUI](https://microsoft.github.io/microsoft-ui-xaml/)) is the native user experience (UX) framework for both Windows Desktop and UWP applications. WinUI is the path forward for all native Windows apps. React Native for Windows is layered on top of WinUI.
+
+[WinUI 3](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) is the next version of WinUI, completely decoupled from the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/) to enable faster shipping of UI features, more backward compatability and to simplify building of version adaptive apps. We are working in close alignment to add WinUI 3 support for React Native for Windows in order to ensure we are always tracking against the latest in native Windows platform. If you are curious, you can try out a preview of this work using the optional flag `--useWinUI3` when running `react-native-windows-init`. You can read more about this effort [here](https://microsoft.github.io/react-native-windows/docs/winui3).
```

Comment:
```suggestion
[WinUI 3](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) is the next version of WinUI, completely decoupled from the [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/) to enable faster shipping of UI features, more backward compatability and to simplify building of version adaptive apps. We are working in close alignment to add WinUI 3 support for React Native for Windows in order to ensure we are always tracking against the latest in native Windows platform. If you are curious, you can try out an experimental preview of this work using the optional flag `--useWinUI3` when running `react-native-windows-init`. You can read more about this effort [here](https://microsoft.github.io/react-native-windows/docs/winui3).
```

Reply:
Diff doesn't read easily, but I replaced with "a preview" with "an experimental preview" to emphasize that we may not be ready for a ton of traffic on this yet.